autoload/neomake/makers/ft/python.vim

```diff
@@ -19,6 +19,10 @@ function! neomake#makers#ft#python#EnabledMakers()
         call add(makers, 'pylint')  " Last because it is the slowest
     endif
 
+    if neomake#utils#Exists('vulture')
```

Comment:
But is probably better to not enable it by default, given it's false positives, e.g. on class attributes / module functions etc.